tests/phpunit/MapsMapperTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Maps\Test;
+
+use MapsMapper;
+
+/**
+ * @covers MapsMapper
+ *
+ * @since 3.6
+ *
+ * @group Maps
+ *
+ * @licence GNU GPL v2+
+ * @author Peter Grassberger < petertheone@gmail.com >
+ */
+class MapsMapperTest extends \PHPUnit_Framework_TestCase {
+
+    public function imageUrlProvider() {
+        return [
+            ['markerImage.png', 'markerImage.png'],
+            ['/w/images/c/ce/Green_marker.png', '/w/images/c/ce/Green_marker.png'],
+            ['//semantic-mediawiki.org/w/images/c/ce/Green_marker.png', '//semantic-mediawiki.org/w/images/c/ce/Green_marker.png'],
+            ['Cat2.jpg', 'Cat2.jpg'],
+        ];
+    }
+
+    /**
+     * Tests MapsMapperTest::getFileUrl()
```

Comment:
I think that part is superfluous. And when using it, I'd use a @covers tag as well, so its easier to detect when its wrong. Not important tho